### PR TITLE
Add a context menu entry if a link is selected

### LINKED ISSUE #55: Context menu entry for link
User:
1. Sees link.
2. Right-click on link.
3. Choose "Generate QR code from link".
-> A QR code should be generated from the link, without the need to actually open/visit the site.

This depends on/extends https://github.com/rugk/offline-qr-code/issues/38.

### PR DESCRIPTION
**Pull request by ENT8R**
This PR fixes #55
screenshot 1

### CODE REVIEW

**Comment by ENT8R** [comment]
> if I actually both select something and right-click on a link? So e.g. select a part of an URL and right-click on it

Then both actions are shown. Should this be handled differently?

**Comment by ENT8R** [comment]
It looks then like this:
screenshot 2

**Comment by rugk** [comment]
> Then both actions are shown. Should this be handled differently?

Ah, no, now that I am seeing that, I think this is the best thing, it can do. :smile:

File: src/_locales/de/messages.json [hunk 1]
```diff
@@ -68,11 +68,15 @@
   // context menu
   "contextMenuItemConvertSelection": {
     "message": "QR-Code aus Auswahl",
-    "description": "The context menu entry shown for generating QR codes from a slection."
+    "description": "The context menu entry shown for generating QR codes from a selection."
+  },
+  "contextMenuItemConvertLinkSelection": {
+    "message": "QR-Code aus Link erstellen",
```
Comment (rugk) [inline src/_locales/de/messages.json]: As with the English translation, don't include the word "erstellen" here. It is obvious this add-on does not destroy QR codes.

**Comment by rugk** [comment]
Thanks, it all works great.

**Pull request merged**